drivers/sqljs-driver/build.gradle

```diff
@@ -13,16 +13,24 @@ kotlin {
       }
     }
   }
-}
 
-dependencies {
-  api projects.runtime
-  implementation libs.kotlin.coroutines.core
-  implementation libs.stately.concurrency
+  sourceSets {
+    jsMain {
```

Comment:
I don't think it makes a difference here, but I've been using commonMain for these deps even though there's only one target.